pyaerocom/io/cams2_83/obs.py

```diff
@@ -71,4 +74,29 @@ def read_csv(
     if (df.conc <= 0).any():
         logger.warning("found negative obs")
         df = df[df.conc > 0]
+    metadata_file_path = Path(path).parent.parent / DEFAULT_METADATA_NAME
+    if metadata_file_path.is_file():
+        df_metadata = read_metadata(metadata_file_path)
+        if not df_metadata.empty:
+            df = df.merge(df_metadata, on="station", how="left")
+            return df["station lat lon alt time poll conc station_type".split()]
+        else:
+            logger.warning(f"Empty metadata from {metadata_file_path}")
+    else:
+        logger.warning(f"Metadata file {metadata_file_path} does not exist")
     return df["station lat lon alt time poll conc".split()]
+
+
+def read_metadata(path: str | Path) -> pd.DataFrame:
+    df = pd.DataFrame()
+    try:
+        df = pd.read_csv(
+            path,
+            sep=",",
+            header=0,
+            names="station station_type".split(),
+            usecols=[0, 2],
+        )
+    except ParserError as e:
+        logger.warning(f"Invalid metadata file {path}, {e}")
```

Comment:
the empty data frame has columns, so `read_csv` is going to crash at
``` python
    return df["station lat lon alt time poll conc".split()]
```
I think that it is better to not mask the `ParserError` or raise a custom error.

Reply:
oh I thought it would be returned empty in this case   
originally the try/except block was outside the read_metadata function... 
I moved it here because I thought it was cleaner but I'll see how to improve this part

Reply:
maybe I'm wrong, will double check